libdevice/fallback-cassert.cpp

```diff
@@ -114,9 +114,4 @@ DEVICE_EXTERN_C void __devicelib_assert_fail(const char *expr, const char *file,
   __assertfail(expr, file, line, func, 1);
 }
 
-DEVICE_EXTERN_C void _wassert(const char *_Message, const char *_File,
```

Comment:
I feel like this is a change that can be merged and submitted separately. `_wassert` is a wrapper for MSVC's `assert` implementation to redirect it to ours, so it really shouldn't be implemented in `fallback` library

Reply:
I don't even know if it's correct, I just hit a build error on windows about `_wassert` defined twice, probably it works today because they're all weak symbols but I remove that as part of this PR.